ledger-core/network/gateway/bootstrap/requester.go

```diff
@@ -205,6 +248,14 @@ func (ac *requester) Bootstrap(ctx context.Context, permit *packet.Permit, candi
 		return respData, throw.New("Bootstrap got UpdateSchedule")
 	case packet.Reject:
 		return respData, throw.New("Bootstrap request rejected")
+	case packet.Retry:
+		time.Sleep(time.Second)
```

Comment:
this should be configurable?

Reply:
probably, would better to use synckit.Backoff here too

Reply:
or at least add a TODO + JIRA pls

Reply:
fixed